libraries/botbuilder-dialogs-adaptive/src/actions/setProperty.ts

```diff
@@ -127,6 +127,6 @@ export class SetProperty<O extends object = {}> extends Dialog<O> implements Set
      * @returns A `string` representing the compute Id.
      */
     protected onComputeId(): string {
-        return `SetProperty[${this.value.toString()}]`;
+        return `SetProperty[${this.property.toString()}]`;
```

Comment:
Note: for [parity w/ .NET](https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/SetProperty.cs#L112)